arm/Microsoft.ApiManagement/serviceResources/apis/deploy.bicep

```diff
@@ -94,8 +94,8 @@ param value string = ''
 @description('Optional. Criteria to limit import of WSDL to a subset of the document.')
 param wsdlSelector object = {}
 
-var apiVersionSetNotEmpty = (!empty(apiVersionSet))
-var apiVersionSetName = (apiVersionSetNotEmpty ? apiVersionSet.name : 'default')
+var apiVersionSetNotEmpty = !empty(apiVersionSet)
+var apiVersionSetName = apiVersionSetNotEmpty ? apiVersionSet.name : 'default'
```

Comment:
```suggestion
var apiVersionSetName = !empty(apiVersionSet) ? apiVersionSet.name : 'default'
```

Reply:
I see that the apiVersionSetNotEmpty is also referenced more places, so maybe ok to keep as is. But not sure why.

Reply:
done